src/Friends/OrganizationsTrait.php

```diff
@@ -22,4 +22,13 @@ public function getOrganizations()
         }
         return $orgs;
     }
+
+    /**
+     * @return SiteOrganizationMemberships|UserOrganizationMemberships
+     * @deprecated 1.0.1 Please use getOrganizationMemberships() instead.
```

Comment:
Makes sense to me. Just need to remember to fully remove whenever we cut the next big version.